### PR TITLE
Heatmap and Contour docs

### PR DESCRIPTION
**Pull request by priyatharsan**
related to: https://github.com/plotly/plotly.github.io/issues/373
and https://github.com/plotly/plotly.github.io/issues/378
- [x] Heatmaps
- [x] Contour Plots
- [x] 2D Contour Histogram Plots

### CODE REVIEW

**Comment by priyatharsan** [comment]
@cldougl just wondering for the title 2d Contour plot doc, whether it should be 2d Histogram Contour (as mentioned in https://github.com/plotly/plotly.github.io/issues/378) or 2D Contour Histogram?
capture

File: _posts/scientific/2016-09-27-how-to-make-a-2d-histogram-contour-plot.md [hunk 1]
```diff
@@ -1,66 +1,59 @@
 ---
 layout: tutorial-single_layout
-title: How to Make a 2D Histogram Contour Plot in Chart Studio
-subtitle: Create a plot that visualizes a bivariate distribution with contour lines.
+title: 2D Contour Histogram Plots in Chart Studio
+subtitle: A type of contour plot that visualizes a bivariate distribution with contour lines.
 permalink: /make-a-2d-histogram-contour-plot/
-imageurl: ../static/images/create-contour-plot.md/2d contour thumb.png
+imageurl: ../static/images/create-2d-hstogram-contour-plot/2d-contour-thumb.png
```
Comment (ashmpace) [inline _posts/scientific/2016-09-27-how-to-make-a-2d-histogram-contour-plot.md]: Just noting "/create-2d-**histogram**-contour-plot" here.

File: _posts/scientific/2016-09-27-how-to-make-a-2d-histogram-contour-plot.md [hunk 2]
```diff
@@ -1,66 +1,59 @@
...
 
 steps:
  - title: Try an Example
    sub-steps:
-    - copy: "This 2 dimensional histogram compares distributions across two variables in a contour plot fashion and the distribution is depicted by a colorscale."
-    - copy: "After selecting '2D Histogram Contour Plot' under 'Chart Type', you can check out an example before adding your own data. Clicking the 'try an example' button will show what a sample chart looks like after adding data and playing with the style. You'll also see what values and style attributes were selected for this specific plot, as well as the end result."
-      img: "![Example after](../static/images/create-contour-plot.md/example contour.png)"
-    - copy: "You can also use the data featured in this tutorial by clicking on 'Open This Data in Plotly' on the left-hand side. It'll open in the workspace."
-      img: "![Open data](../static/images/enter-data-in-the-grid/open-this-data.png)"
+    - copy: "2D histogram compares distributions across two variables in a contour plot fashion and the distribution is depicted by a colorscale."
```
Comment (ashmpace) [inline _posts/scientific/2016-09-27-how-to-make-a-2d-histogram-contour-plot.md]: Suggestion: "This 2D histogram uses a colorscale to depict distribution across two variables in a contour plot."

**Comment by ashmpace** [comment]
Made copy edits and a couple suggestion in comments. Otherwise all good 👍

File: _posts/basic/2016-07-08-how-to-create-a-contour-plot.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+ - title: Add Your Data to Plotly
+   sub-steps:
+    - copy: "Head to Plotly’s [Chart Studio](https://plot.ly/create/) and add your data. You have the option of typing directly in the grid, uploading your file, or entering the URL of an online dataset. Plotly accepts .xls, .xlsx, or .csv files. For more information on how to enter your data, see [this](https://help.plot.ly/add-data-to-the-plotly-grid/) tutorial."
+
+ - title: Create a Chart
+   sub-steps:
+    - copy: "After adding data, go to the 'Traces' section under the 'Structure' menu on the left-hand side. Choose the 'Type' of trace, then choose 'Contour' under 'Simple' chart type."
+      img: "![Choose chart](../static/images/create-contour-plot/contour-choose-chart.png)"
+    - copy: "Next, fill out the necessary 'X', 'Y' and 'Z' values from their respective dropdown menus. This will create a contour, as seen below. In this example plot, we'll add all columns of our dataset in the 'Z' matrix, since there is no 'X' and 'Y' data."
```
Comment (cldougl) [inline _posts/basic/2016-07-08-how-to-create-a-contour-plot.md]: I would reword this to mention that X and Y are optional for this plot:

In this example, our data was formatted as a Z matrix so we will add all columns of our dataset to the 'Z' dropdown. We do not have specified 'X' or 'Y' data in this example, so we will leave those dropdown fields empty.

File: _posts/basic/2016-07-08-how-to-create-a-contour-plot.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+      img: "![Adding axes](../static/images/create-contour-plot/contour-values.png)"
+
+ - title: Style a Chart
```
Comment (cldougl) [inline _posts/basic/2016-07-08-how-to-create-a-contour-plot.md]: I think the general section should come first as that's how the menus are listed

File: _posts/scientific/2016-09-27-how-to-make-a-2d-histogram-contour-plot.md [hunk 5]
```diff
@@ -1,66 +1,59 @@
...
-      img: "![Annotation](../static/images/create-contour-plot.md/contour annotation.png)"
-    - copy: "Another option is to visit the 'Layout' section under STYLE, click on 'Title and Fonts' and enter your title in the box, as shown below."
-      img: "![Type in layout panel](../static/images/create-contour-plot.md/contour title tab.png)"
+    - copy: "The 'Style' menu displays many options to modify characteristics of the overall chart layout or the individual traces. To see more options about styling the chart visit the [style and layout](https://help.plot.ly/tutorials/#layout) section of the Chart Studio documentation."
+    - copy: "Use the 'Traces' section under the 'Style' menu to change the properties of the traces, such as histogram and binning attributes, color gradients, contours and line attributes."
+      img: "![Colourscale](../static/images/create-2d-contour-histogram-plot/contour2D-properties.png)"
+    - copy: "Use the 'General' section under the 'Style' menu to set the plot title and change the layout background, margin color and font styles."
```
Comment (cldougl) [inline _posts/scientific/2016-09-27-how-to-make-a-2d-histogram-contour-plot.md]: Same comment as above-- I'd suggest discussing general first then traces

File: _posts/scientific/2016-09-27-how-to-make-a-2d-histogram-contour-plot.md (hunk 2 above)
Comment (priyatharsan) [reply _posts/scientific/2016-09-27-how-to-make-a-2d-histogram-contour-plot.md]: 👍

File: _posts/scientific/2016-09-27-how-to-make-a-2d-histogram-contour-plot.md (hunk 1 above)
Comment (priyatharsan) [reply _posts/scientific/2016-09-27-how-to-make-a-2d-histogram-contour-plot.md]: gotcha 👍

File: _posts/basic/2016-07-08-how-to-create-a-contour-plot.md (hunk 3 above)
Comment (priyatharsan) [reply _posts/basic/2016-07-08-how-to-create-a-contour-plot.md]: 👍

File: _posts/basic/2016-07-08-how-to-create-a-contour-plot.md (hunk 4 above)
Comment (priyatharsan) [reply _posts/basic/2016-07-08-how-to-create-a-contour-plot.md]: 👍

File: _posts/scientific/2016-09-27-how-to-make-a-2d-histogram-contour-plot.md (hunk 5 above)
Comment (priyatharsan) [reply _posts/scientific/2016-09-27-how-to-make-a-2d-histogram-contour-plot.md]: 👍

**Comment by priyatharsan** [comment]
tysm! @ashmpace @cldougl
